miden-lib/asm/miden/account.masm

```diff
@@ -101,6 +101,42 @@ export.set_item
     # => [R', V]
 end
 
+#! Gets a map item from the account storage. Panics if the index for the map is out of 
+#! bounds, means >255.
+#!
+#! Stack: [index, KEY]
+#! Output: [VALUE]
+#!
+#! - index is the index of the map where the KEY VALUE should be read.
+#! - VALUE is the value of the item.
+export.get_map_item
+    syscall.get_account_map_item
+    # => [VALUE]
+
+    # prepare stack for return
+    movup.5 drop
+    # => [VALUE, 0]
+end
+
+#! Sets a map item in the account storage. Panics if the index for the map is out of 
+#! bounds, means >255.
+#!
+#! Stack: [index, KEY, VALUE]
+#! Output: [OLD_MAP_ROOT, OLD_MAP_VALUE]
+#!
+#! - index is the index of the map where the KEY VALUE should be set.
+#! - VALUE is the value to set at KEY.
+#! - OLD_MAP_ROOT is the old map root.
+#! - OLD_MAP_VALUE is the old value at KEY.
+export.set_map_item
+    syscall.set_account_map_item
+    # => [OLD_MAP_ROOT, OLD_MAP_VALUE]
+
+    # prepare stack for return
+    push.0 movdn.9
+    # => [OLD_MAP_ROOT, OLD_MAP_VALUE, 0]
```

Comment:
The stack cleanup seems reverted. The `set_account_map_item` returns:

> [OLD_MAP_ROOT, OLD_MAP_VALUE, 0]

So the prepare should instead `movup.9 drop`

Reply:
I don't think so, we want to receive 9 items and return 9 items, right? If that is the case we need to add a `0`